internal/ingest/ingest_test.go

```diff
@@ -66,10 +66,10 @@ func TestSubscribe(t *testing.T) {
 	// Test with two random advertisement publications.
 	_, mhs := publishRandomAdv(t, i, lph, lp, lsys, false)
 	// Check that the mhs have been indexed correctly.
-	i.checkMhsIndexed(t, lph.ID(), mhs)
+	i.checkMhsIndexedEventually(t, lph.ID(), mhs, 2*time.Second, 10*time.Second)
```

Comment:
The timeouts should be defined as consts or made configurable.

If they are always the same for each call to `i.checkMhsIndexedEventually`, then set them when the `Ingester` is instantiated.